test/format/snap.c

```diff
@@ -162,7 +162,9 @@ snap_track(TINFO *tinfo, thread_op op)
             snap->kdata = drealloc(snap->kdata, ip->size);
             snap->kmemsize = ip->size;
         }
-        memcpy(snap->kdata, ip->data, snap->ksize = ip->size);
+        snap->ksize = ip->size;
+        if (ip->size)
```

Comment:
Maybe use `ip->size > 0` here to be consistent with line 177?

Reply:
Consistency is good! Well spotted - I'll make the change.